src/io/aisetfreq.c

```diff
@@ -1,6 +1,8 @@
 #include "PR/rcp.h"
 #include "PR/ultraerror.h"
 #include "../os/osint.h"
+//not sure if this should be here
+extern s32 osViClock;
```

Comment:
```suggestion
// TODO: not sure if this should be here
extern s32 osViClock;
```

This way is easier to find stuff like this